src/lib/OpenEXRCore/internal_dwa_decoder.h

```diff
@@ -215,7 +271,7 @@ LossyDctDecoder_base_construct (
     d->_toLinear      = toLinear;
     d->_width         = width;
     d->_height        = height;
-    if (d->_toLinear == NULL) d->_toLinear = dwaCompressorNoOp;
+    //if (d->_toLinear == NULL) d->_toLinear = dwaCompressorNoOp;
```

Comment:
is this commenting out intentional

Reply:
ah, yeah, I missed fully removing that. I also removed the no-op table in favor of an outer loop conditional to dispatch to do nothing instead of always looking through an identity table.